husky_description/urdf/description.xacro

```diff
@@ -31,7 +31,7 @@ ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSI
   <xacro:arg name="kinect_enabled" default="false" />
   <xacro:arg name="robotiq_grippers_enabled" default="$(optenv ROBOTIQ_GRIPPERS_ENABLED false)" />
  
-  <xacro:include filename="$(find husky_description)/urdf/husky.urdf.xacro" />
+  <xacro:include filename="$(find husky_description)/urdf/dual_arm_husky.urdf.xacro" />
```

Comment:
Hi @wxmerkt,

My apologies - I was looking at the wrong branch. It's been many months since I've done this ;p 

It does appear that this change is correct and we can go forward with it.